ports/unix/variants/DEV/mpconfigvariant.mk

```diff
@@ -0,0 +1,20 @@
+PROG ?= micropython
```

Comment:
So the default `make` will now be the dev build, with executable named `micropython`... then the original base variant (got with `make` currently) is gone?  Maybe we need a STANDARD variant (lean but featureful) and a DEV variant (all features, including settrace?)?

Reply:
This was specifically to make it match the current behavior -- the default target produces the `micropython` binary.

Reply:
Ok, but the default variant is now called "DEV", right?  As below, maybe we need a BASE/STANDARD build and a DEV/FULL build.